tests/func/test_remote.py

```diff
@@ -162,15 +162,16 @@ def test(self):
 
 def test_large_dir_progress(repo_dir, dvc_repo):
     from dvc.utils import LARGE_DIR_SIZE
-    from dvc.progress import progress
+    from dvc.progress import Tqdm
 
     # Create a "large dir"
     for i in range(LARGE_DIR_SIZE + 1):
         repo_dir.create(os.path.join("gen", "{}.txt".format(i)), str(i))
 
-    with patch.object(progress, "update_target") as update_target:
+    with patch.object(Tqdm, "truncate") as truncate:
+        assert not truncate.called
         dvc_repo.add("gen")
-        assert update_target.called
+        assert truncate.called
```

Comment:
Why we check for truncate here? We should check for a method that outputs a progress bar I guess.

Reply:
yes it's just an update of the previous test stub. `truncate` is a classmethod so was the easiest thing for me to patch

Reply:
Can we patch some `.update*`method instead?